### DISCUSSION TITLE
Any documentation available for SceneView

### DISCUSSION THREAD

**Discussion by Hanif-MI** [sceneview/sceneview#490] [Q&A]
@ThomasGorisse sir, Any documentation available for ScenceView

**Comment by Danwolve98** [comment]
U can check the api
https://sceneview.github.io/api/sceneview-android/index.html

**Comment by Hanif-MI** [comment]
I have checked this document but I want a document that gives me the proper implementation steps of SceneView.

**Comment by ThomasGorisse** [comment]
Yes! SceneView now has extensive documentation:

- **Docs site**: [sceneview.github.io](https://sceneview.github.io) — quickstart, API reference, cheatsheet, migration guide
- **llms.txt**: Machine-readable API reference at the repo root — complete composable signatures, node types, threading rules, and patterns
- **Samples**: `samples/android-demo` has 19 working demos covering all features
- **Recipes**: `samples/recipes/` has markdown code recipes for common tasks (model viewer, AR, physics, geometry, text)
- **MCP server**: `npx sceneview-mcp` lets AI assistants generate correct SceneView code

Latest version: `io.github.sceneview:sceneview:4.0.0-rc.1`
